File: 2418-sort-the-people/2418-sort-the-people.java
class Solution {
    public String[] sortPeople(String[] names, int[] heights) {
        HashMap<Integer,String> map = new HashMap<Integer,String>();
        for(int i=0;i<heights.length;i++)
        {
            map.put(heights[i],names[i]);
        }
        Arrays.sort(heights);
        String[] res = new String[heights.length];
        int index=0;
        for(int j=heights.length-1;j>=0;j--)
        {
            res[index]=map.get(heights[j]);
            index++;
        }
        return res;
    }
}

File: 2418-sort-the-people/2418-sort-the-people.py
class Solution:
    def sortPeople(self, names: List[str], heights: List[int]) -> List[str]:
        sorted_heights = sorted(heights,reverse=True)
        list2=[]
        for height in sorted_heights:
            z = heights.index(height)
            list2.append(names[z])
        return list2